### ISSUE TITLE
TAS_L z_boom & z_stake flipped in 2007/2008 ?

### ISSUE THREAD

**Issue by mvantiggelen** [GEUS-Glaciology-and-Climate/PROMICE-AWS-data-issues#129]
It seems that the **z_stake** and **z_boom_u** variables from the SR50 measurements are flipped in the TAS_L data during 2007/2008.
file: _TAS_L_hour.csv_ downloaded from Dataverse V12 on 15-12-2023

![image](https://github.com/GEUS-Glaciology-and-Climate/PROMICE-AWS-data-issues/assets/39654529/38237277-9bca-47b0-a26f-c75ab827b724)

**Comment by BaptisteVandecrux** [comment]
Thanks Maurice for the report!

Indeed installation was done on bare ice so the initial values of z_boom_u should be the standard station height ~2.6 m.

I have swapped the two channels in our processing of the 2007-2008 logger file.

I can see that by 2010-2011, the swap in wiring was fixed: z_boom_u starting at ~2.5 m after station maintenance and z_stake getting larger than the station height in 2011. But I cannot see if the fix happened in summer 2008, 2009 or 2010. Anyway the data quality during 2008-2010 is quite poor so we'll need to filter out those periods anyway.

I'll make an update when it is fixed on a new Dataverse release.
